stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -65,6 +67,34 @@
     private static final SSLSocketFactory SSL_SOCKET_FACTORY = new StripeSSLSocketFactory();
 
     /**
+     * Create a {@link Source} using the input {@link SourceParams}.
+     *
+     * @param sourceParams a {@link SourceParams} object with {@link Source} creation params
+     * @param publishableKey an API key
+     * @return a {@link Source} if one could be created from the input params,
+     * or {@code null} if not
+     * @throws AuthenticationException if there is a problem authenticating to the Stripe API
+     * @throws InvalidRequestException if one or more of the parameters is incorrect
+     * @throws APIConnectionException if there is a problem connecting to the Stripe API
+     * @throws CardException if there is a problem with the card information
+     * @throws APIException for unknown Stripe API errors. These should be rare.
+     */
+    @Nullable
+    public static Source createSourceOnServer(
```

Comment:
This is a simpler thing to call than how we currently create card and bank tokens, and all the fun diffs about JSON that were previously done are why I'm able to do it very succinctly here. The Card methods are left alone so we don't break back-compatibility.